src/id-card/index.html

```diff
@@ -223,7 +223,7 @@
                     },
                     saveIdCard: function () {
                         console.log('saveIdCard');
-                        this.idCardToIssue['revocationKey'] = this.idCardToIssue.number;
+                        this.idCardToIssue['revocationKey'] = luxon.DateTime.now().toUTC().toMillis().toString();
```

Comment:
I had to change that because is a bug(?) in IRMA that prevents issuing the same credential with the same `revocationKey`! So that's an issue when we refreshing the card! For now, I generate a unique value in every issuance.